backend/tests/api/instances/[id]/get.test.ts

```diff
@@ -7,3 +8,15 @@ Deno.test('GET /api/instances/unknown-id that it returns 404!', async () => {
     .expect('Content-Type', /text/)
     .expect(404);
 });
+
+Deno.test('GET /api/instances/id that it returns the id of the created instance', async () => {
+  const jimmi = await createJimmi();
+  await superdeno(app)
+    .get(`/api/instances/${jimmi.id}`)
+    .expect('Content-Type', /json/)
+    .expect(200)
+    .then((res) => {
+      if (!(Object.keys(res.body).includes('id'))) throw new Error('res.body should include id');
```

Comment:
Is there a reason not to use `in`?
```suggestion
      if (!('id' in res.body) throw new Error('res.body should include id');
```